.phpcs.xml.dist

```diff
@@ -29,15 +16,14 @@
 		</properties>
 	</rule>
 
-	<!-- Check all PHP files in directory tree by default. -->
-	<arg name="extensions" value="php"/>
-	<file>.</file>
-
-	<!-- Show sniff codes in all reports -->
-	<arg value="s"/>
-
-	<arg name="colors"/>
-
-	<exclude-pattern>/node_modules/*</exclude-pattern>
-	<exclude-pattern>/vendor/*</exclude-pattern>
+	<!-- For the coding-standards package, we want to follow PHPCS convention, so excluding some WP rules. -->
+	<rule ref="WordPress.Files.FileName">
+    	<exclude-pattern>/packages/coding-standards/*</exclude-pattern>
```

Comment:
All these indented lines here and below use spaces, not tabs.

Reply:
2a7128a5e